src/client/components/ProfileCardComponent/ProfileCardComponent.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './ProfileCardComponent.styles.css';
+import iconLinkedin from '../../assets/images/icons/linkedin.svg';
+import iconWebsite from '../../assets/images/icons/website.svg';
+import iconGithub from '../../assets/images/icons/github.svg';
+import unknownUser from '../../assets/images/portraits/unknown-black.jpg';
+
+export const ProfileCardComponent = ({
+  profile_image_url,
```

Comment:
```suggestion
  profileImageUrl,
```

Reply:
Bejamin, but in our database the names are given with the underscores and therefore I decided to go with the names that match the database